src/App.jsx

```diff
@@ -1,10 +1,24 @@
 import React, { Component } from 'react';
+import { BrowserRouter as Router, Route, Link, Switch } from 'react-router-dom';
+import login from './components/Login';
+import PageNotFound from './components/NotFoundComponent';
+import Navbar from './components/Navbar';
+import home from './components/home';
```

Comment:
add history as a dependancy and `import { createBrowserHistory } from 'history'` then pass it as props to Router

Reply:
@koechkevin I have done that